projects/plugins/social/src/class-jetpack-social.php

```diff
@@ -169,6 +168,7 @@ public function initial_state() {
 			),
 			'sharesData'      => ! is_wp_error( $shares ) ? $shares : null,
 			'showNudge'       => $show_nudge,
+			'siteRawUrl'      => ( new Status() )->get_site_suffix(),
```

Comment:
There's an equivalent to this in the `Jetpack_Editor_Initial_State` called `siteFragment`. Which can be retrieved with `getSiteFragment`. 

Then this is [called `rawUrl` in `Initial_State`](https://github.com/Automattic/jetpack/blob/2447e267e40f2d133a5c15c2320b776e4c395ec8/projects/plugins/jetpack/_inc/lib/admin-pages/class-jetpack-redux-state-helper.php#L138) for Jetpack. Maybe not right now, but what do you thinking about aligning the naming of these properties, so we could have a shared store for initial state?